lib/IRGen/GenFunc.cpp

```diff
@@ -2400,7 +2400,8 @@ llvm::Function *IRGenFunction::getOrCreateResumePrjFn() {
         }
         // TODO: remove this once all platforms support lowering the intrinsic.
         // At the time of this writing only arm64 supports it.
-        if (IGM.TargetInfo.canUseSwiftAsyncContextAddrIntrinsic()) {
+        if (IGM.getOptions().UseAsyncLowering &&
```

Comment:
Same here. We already support/emit `swift_async_context` in tree. This does not need to be condititionalized on `UseAsyncLowering`.

Reply:
Removed.